MainRepository/Program.cs

```diff
@@ -1,7 +1,21 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿/// <summary>Implementation of Anagram</summary>

```

Comment:
Refer [this](https://github.com/karthick-1205/Assignments/pull/16#discussion_r1381194663) and update the comment.